multicluster/controllers/multicluster/stale_controller.go

```diff
@@ -289,7 +348,6 @@ func (c *StaleResCleanupController) Run(stopCh <-chan struct{}) {
 		go wait.Until(c.runWorker, time.Second, stopCh)
 	}
 
-	<-stopCh
```

Comment:
Removed by accident?

Reply:
The stop ch was removed because an ticker was added. When removed the ticker, forget to add it back. Added now.